src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/RefreshEmployerLevyData/RefreshEmployerLevyDataCommandHandler.cs

```diff
@@ -154,16 +166,61 @@ private bool IsEndOfYearAdjustment(DasDeclaration dasDeclaration)
                     x.PayrollMonth
                 }).Distinct();
 
-            var tasks = periodsChanged.Select(x => CreateDeclarationUpdatedEvent(hashedAccountId, x.PayrollYear, x.PayrollMonth));
+            var tasks = periodsChanged.Select(x => CreateDeclarationUpdatedGenericEvent(hashedAccountId, x.PayrollYear, x.PayrollMonth));
             await Task.WhenAll(tasks);
         }
 
-        private Task CreateDeclarationUpdatedEvent(string hashedAccountId, string payrollYear, short? payrollMonth)
+        private async Task CreateDeclarationUpdatedGenericEvent(string hashedAccountId, string payrollYear, short? payrollMonth)
         {
             var declarationUpdatedEvent = _levyEventFactory.CreateDeclarationUpdatedEvent(hashedAccountId, payrollYear, payrollMonth);
             var genericEvent = _genericEventFactory.Create(declarationUpdatedEvent);
 
-            return _mediator.SendAsync(new PublishGenericEventCommand { Event = genericEvent });
+            await _mediator.SendAsync(new PublishGenericEventCommand { Event = genericEvent });
+            await PublishDeclarationUpdatedEventMessage(hashedAccountId, payrollYear, payrollMonth);
+        }
+
+        private async Task PublishDeclarationUpdatedEventMessage(string hashedAccountId, string payrollYear, short? payrollMonth)
+        {
+            if (!payrollMonth.HasValue)
+                return;
+
+            try
+            {
+                var accountId = _hashingService.DecodeValue(hashedAccountId);
+                var dasLevyDeclarations = await _dasLevyRepository.GetAccountLevyDeclarations(accountId, payrollYear, payrollMonth.Value);
+                if (!dasLevyDeclarations?.Any() ?? true)
+                {
+                    _logger.Warn($"No levy declarations found for account: {accountId}, payroll year: {payrollYear}, month: {payrollMonth}");
+                    return;
+                }
+                foreach (var levyDeclarationView in dasLevyDeclarations)
+                {
+                    try
+                    {
+                        var levySchemeDeclarationUpdatedEvent =
+                            new LevySchemeDeclarationUpdatedMessage(accountId, null, null);
+                        _mapper.Map(levyDeclarationView, levySchemeDeclarationUpdatedEvent);
+                        await _messagePublisher.PublishAsync(levySchemeDeclarationUpdatedEvent);
+                    }
+                    catch (Exception ex)
+                    {
+                        _logger.Error(ex, $"Error while trying to publish levy scheme declaration changed event. Error: {ex.Message}. Levy data: {JsonConvert.SerializeObject(levyDeclarationView)}");
+                        throw;  //TODO: Not sure if we should rollback the levy processing tx if the publishing fails
```

Comment:
Given this is done as a backend process lets leave in the rethrow and see if it causes major issues as it would be nice to stop everything if events cannot be sent